components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -427,24 +400,16 @@ See [Policy Membership]({{< relref "iam-v2-guide.md#policy-membership" >}}) for
 While we have safeguards to prevent it, it is possible to lock yourself out of Chef Automate.
 If you have root access to the node where Chef Automate is installed, use the following commands to restore admin access:
 
-This command, which is also available on IAM v1, resets the local `admin` user's password and ensures that user is a member of the local `admins` team, which is a permanent member of the Chef-managed `Administrator` policy.
+This command resets the local `admin` user's password and ensures that the user is a member of the local `admins` team, which is a permanent member of the Chef-managed `Administrator` policy.
```

Comment:
is this true?

Reply:
okay yes true, the error message is ` Could not remove members from policy: cannot remove local team: admins from Chef-managed policy: Admin` in the future it would be cool to hide the remove option all together from the ui.